config.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+class day(Enum):
```

Comment:
You don't need to reinvent the wheel. You can use builtin functions to check the day of the week.